buildSrc/src/test/groovy/com/google/firebase/gradle/plugins/publish/PublishingPluginSpec.groovy

```diff
@@ -44,13 +44,15 @@ class PublishingPluginSpec extends Specification {
 	    }
         dependencies {
             <%dependencies.each { println "implementation project(':$it.name')" } %>
+            <%customDependencies.each { println "implementation '$it'" } %>
         }
         '''
         String name
         String group = 'com.example'
         String version = 'undefined'
         String latestReleasedVersion = ''
         Set<Project> projectDependencies = []
+        Set<String> customDependencies = []
```

Comment:
I think "customDependencies" is a bit confusing. In my mind there are "implementation project(...)" dependencies, and "implementation (...)" dependencies, so maybe "nonProjectDependencies", "external dependencies"?

Reply:
Good point, changed to `externalDependencies`

Reply:
Much better, thanks!